test-suite/unit-tests/clib-tests/inteq.ml

```diff
@@ -0,0 +1,14 @@
+open OUnit
```

Comment:
Seems redundant to have "clib-tests" in the pathname when "unit-tests" is above it.

Seems to me the unit test path should generally be parallel to the source path, so unit-tests/clib/XXX.ml is a test for coq/clib/XXX.ml.

On the other hand, having the last part of the file path for the test code match the source will make it a little harder to open the right file (my editor suggests matches).

Reply:
The directory names can (and should) be the same -- I'll make that change. There is trouble if you name the individual files the same as the corresponding Coq sources.